security/pfSense-pkg-snort/files/usr/local/pkg/snort/snort_migrate_config.php

```diff
@@ -232,9 +216,10 @@
 /* element for existing entries into an array. Migrate    */
 /* any existing <address> to the new array structure.     */
 /**********************************************************/
-if (is_array($config['installedpackages']['snortglobal']['whitelist']['item'])) {
-	foreach ($config['installedpackages']['snortglobal']['whitelist']['item'] as &$wlisti) {
-		if (!is_array($wlisti['address']) && !is_array($wlisti['address']['item']) && !empty($wlisti['address'])) {
+if (config_get_path('installedpackages/snortglobal/whitelist/item')) {
+	$a_wlist = config_get_path('installedpackages/snortglobal/whitelist/item', []);
```

Comment:
```suggestion
$a_wlist = config_get_path('installedpackages/snortglobal/whitelist/item', []);
```
I'd recommend just dropping the conditional here